packages/manager/src/components/TagsInput/TagsInput.tsx

```diff
@@ -128,21 +142,43 @@ export const TagsInput = (props: TagsInputProps) => {
         : undefined);
 
   return (
-    <Select
-      creatable
+    <Autocomplete
+      onChange={(_, newValue, reason, details) => {
+        const detailsOption = details?.option;
+        if (
+          reason === 'selectOption' &&
+          detailsOption?.label.includes(`Create "${detailsOption?.value}"`)
+        ) {
+          createTag(detailsOption.value);
+        } else {
+          setErrors([]);
+          onChange(newValue);
+        }
+      }}
+      renderTags={(tagValue, getTagProps) => {
+        return tagValue.map((option, index) => (
+          <Chip
+            {...getTagProps({ index })}
+            deleteIcon={<CloseIcon />}
+            key={index}
+            label={option.label}
+            onDelete={() => handleRemoveOption(option)}
+          />
+        ));
+      }}
+      autoHighlight
+      clearOnBlur
+      disableCloseOnSelect={false}
+      disabled={disabled}
       errorText={error}
-      hideLabel={hideLabel}
-      isDisabled={disabled}
-      isMulti={true}
+      filterOptions={filterOptions}
+      isOptionEqualToValue={(option, value) => option.value === value.value}
       label={label || 'Add Tags'}
-      menuPlacement={menuPlacement}
-      name={name}
-      noMarginTop={noMarginTop}
-      noOptionsMessage={getEmptyMessage}
-      onChange={onChange}
-      onCreateOption={createTag}
+      multiple
+      noOptionsText={'No results'}
       options={accountTagItems}
-      placeholder={'Type to choose or create a tag.'}
+      placeholder={value.length === 0 ? 'Type to choose or create a tag.' : ' '}
```

Comment:
Any reason for this space?

```suggestion
      placeholder={value.length === 0 ? 'Type to choose or create a tag.' : ''}
```

Reply:
We can remove that space.

Reply:
@hkhalil-akamai what is it with the **"Coverage Report"**?